### DISCUSSION TITLE
Regex a prefix string with fixed length

### DISCUSSION THREAD

**Discussion by tarikrital** [pillarjs/path-to-regexp#248] [Q&A]
it is mentioned in vuejs router doc that route path depends in your project, thank you

is it possible to apply the failed regex: `^(AB|DE|GH)[a-zA-Z0-9]{15}$` as the successful `path: '/:qrcode(\\d+)` ?

**Comment by blakeembrey** [comment]
Like this? `/:qrcode((?:AB|DE|GH)[a-zA-Z0-9]{15})`
